openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -45,7 +45,7 @@
 from lms.djangoapps.grades.api import CourseGradeFactory
 from lms.djangoapps.verify_student.services import IDVerificationService
 from openedx.core.djangoapps.agreements.api import get_integrity_signature
-from openedx.core.djangoapps.agreements.toggles import is_integrity_signature_enabled
+from openedx.core.djangoapps.agreements.toggles import is_integrity_signature_enabled as _is_integrity_signature_enabled
```

Comment:
this name choice confused me because I expected _is_integrity_signature_enabled to be the class internal details version of is_integrity_signature_enabled but it turned out to be an external function

I think you could technically get away without it and write
`def is_i_enabled(self): return is_i_enabled(self.course_key)`
just because one is on the class and the other isn't. But that's much worse.

maybe just import as something like integrity_signature_toggle? or from openedx.core.djangoapps.agreements import toggles and then toggles.is_i_enabled